### DISCUSSION TITLE
Is Zig integration working fine for everyone?

### DISCUSSION THREAD

**Discussion by alosarjos** [LazyVim/LazyVim#4739] [Q&A]
I just tried with a fresh LazyVim install, enabled the new Zig lang extra, but as soon as I start editing some code I get this error messages regarding nvim-cmp and nvim-snippets. I haven't found any info regarding this

![Captura desde 2024-11-09 09-51-47](https://github.com/user-attachments/assets/f7f8746b-9074-4752-9c0d-510f13a33f55)

**Comment by mconrado** [comment]
Probably due new update (13.0.0 breaking changes), this update was released yesterday and I have similar issues.

**Reply by alosarjos** [reply]
Pretty annoying its propping all the time :S

**Comment by alosarjos** [comment]
Made the screenshot in English in case that helps

![imagen](https://github.com/user-attachments/assets/9b5fc578-305c-4b8d-91ab-b780cd1b3bd6)

**Reply by mconrado** [reply]
> Made the screenshot in English in case that helps
> 
> ![imagen](https://private-user-images.githubusercontent.com/29999427/384601196-9b5fc578-305c-4b8d-91ab-b780cd1b3bd6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzExNjM4NTAsIm5iZiI6MTczMTE2MzU1MCwicGF0aCI6Ii8yOTk5OTQyNy8zODQ2MDExOTYtOWI1ZmM1NzgtMzA1Yy00YjhkLTkxYWItYjc4MGNkMWIzYmQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA5VDE0NDU1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmNWFkMTJjNDVhNDA3OTFiZDM1NGJlMmJjMWVjNGUxY2QyOGFjMjcwZTBlMWNkMzc1YjMyMzg1MTkxMzk2NTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1lyJgJ3uoYAwdsSoglzEOf7F452Tm_tWPq50FN8_2vQ)

run the LazyVim update, the problem was with snacks.vim. It's solved for me.

**Reply by alosarjos** [reply]
Still getting the error :S


https://github.com/user-attachments/assets/ca31cf93-3deb-4c74-a3f3-dff442b3adeb
